### PR TITLE
switch 429 response headers

### PR DESCRIPTION
**Pull request by Eomm**
Follow #57

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,7 +66,13 @@ fastify.register(require('fastify-rate-limit'), {
   redis: new Redis({ host: '127.0.0.1' }), // default null
   skipOnError: true, // default false
   keyGenerator: function(req) { /* ... */ }, // default (req) => req.raw.ip
-  errorResponseBuilder: function(req, context) { /* ... */}, 
+  errorResponseBuilder: function(req, context) { /* ... */},
+  showHeaders: { // default show all the response headers when rate limit is reached
```
Comment (delvedor) [inline README.md]: How about `sendHeaders` or `displayHeaders`?

File: index.d.ts [hunk 2]
```diff
@@ -12,6 +12,7 @@ declare namespace fastifyRateLimit {
     skipOnError?: boolean;
     keyGenerator?: (req: fastify.FastifyRequest<http.IncomingMessage>) => string | number;
     errorResponseBuilder?: (req: fastify.FastifyRequest<http.IncomingMessage>, context: errorResponseBuilderContext) => object;
+    showHeaders?: object;
```
Comment (delvedor) [inline index.d.ts]: Having an interface here would be better:
```ts
interface ShowHeaders {
  'x-ratelimit-limit'?: boolean,
  'x-ratelimit-remaining'?: boolean,
  'x-ratelimit-reset'?: boolean,
  'retry-after'?: boolean
}
```

File: README.md (hunk 1 above)
Comment (Eomm) [reply README.md]: How sounds `limitedResponseHeaders`?

**Comment by Eomm** [comment]
> I’w call the option addHeaders instead

Done 👍

**Pull request merged**